templates/travis/.travis/performance_test_script.sh.j2

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+if [ "$TRAVIS_EVENT_TYPE" = "cron" ]; then
+  echo "--- Performance Tests ---"
+  pytest -vv -r sx --color=yes --pyargs --durations=0 pulp_rpm.tests.performance
```

Comment:
The `pulp_rpm` needs to be generalized with a jinja template for the plugin name.

Reply:
thanks, I didn't notice it